binaries/apiserver/main.go

```diff
@@ -71,15 +72,15 @@ func main() {
 				sh.endpoint: sh.handler,
 			}
 		},
-		func(fileStore *bundlestore.FileStore, stat stats.StatsReceiver, tmp *temp.TempDir) (*StoreAndHandler, error) {
+		func(fileStore *bundlestore.FileStore, stat stats.StatsReceiver, upTimeReportIntv stats.UpTimeReportIntvl, tmp *temp.TempDir) (*StoreAndHandler, error) {
 			cfg := &bundlestore.GroupcacheConfig{
 				Name:         "apiserver",
 				Memory_bytes: 2 * 1024 * 1024 * 1024, //2GB
 				AddrSelf:     *httpAddr,
 				Endpoint:     "/groupcache",
 				Cluster:      createCluster(),
 			}
-			store, handler, err := bundlestore.MakeGroupcacheStore(fileStore, cfg, stat)
+			store, handler, err := bundlestore.MakeGroupcacheStore(fileStore, cfg, stat, upTimeReportIntv)
```

Comment:
this a a collection interval for a specific set of metrics.  Don't think this belongs in the stat object - different metrics will probably end up with different collection intervals. 

Will update names to Intvl